game_eggs/README.md

```diff
@@ -181,6 +181,7 @@
 * [Onset](steamcmd_servers/onset)
 * [Open Fortress](steamcmd_servers/open_fortress)
 * [Operation Harsh Doorstop](steamcmd_servers/operation_harsh_doorstop)
+* [Palworld(steamcmd_servers/palworld)
```

Comment:
Looks like there's a slight typo that breaks the label on this link.

Reply:
fixed in [39f170f](https://github.com/parkervcp/eggs/pull/2671/commits/39f170fc25fdac41cba04cf3d1912bc8367fc6d1), ty!